src/components/canvas/DataCanvas.ts

```diff
@@ -37,14 +37,5 @@ export default class DataCanvas extends AbstractCanvas {
         },
       ],
     },
-    {
-      category: DataCategories.Custom,
-      nodes: [
-        {
-          name: CommonNodes.Custom,
-          node: Custom,
```

Comment:
So the problem with just removing it here, is that Lancelot uses this big list o actually register the nodes. So removing it means that if you try to add a Custom function node, you get the error `Custom not defined` or something. I'll take a look in the morning, because I did a similar thing for the names of `InModel` and `OutModel` in the `LayersTab` or wherever.

Reply:
Ah yes maybe my solution here is not really good anymore! Do you think we should just filter out `Custom` from the component or should we revert to the way it was prior to my refactoring here?

Reply:
I will just register `CustomNode` in `AbstractCanvas` seperately